src/instrumentation.node.ts

```diff
@@ -7,10 +9,17 @@ import {
   SEMRESATTRS_SERVICE_NAME,
 } from '@opentelemetry/semantic-conventions';
 import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
-import {
-  SentrySpanProcessor,
-  SentryPropagator,
-} from '@sentry/opentelemetry-node';
+import { SentrySpanProcessor, SentryPropagator } from '@sentry/opentelemetry';
+import { SENTRY_CAPTURE_RATE, SENTRY_DSN } from 'sentry.constants.mjs';
+
+init({
+  dsn: SENTRY_DSN,
+  debug: true,
+  tracesSampleRate: SENTRY_CAPTURE_RATE,
+  profilesSampleRate: SENTRY_CAPTURE_RATE,
```

Comment:
no `instrumenter: 'otel'` option in v8

Reply:
yes this was removed, because you no longer need to initialize opentelemetry, the Sentry SDK will do this for you!

so you can remove lines 24 and below.

Reply:
Can i pass any options inside, e.g. some custom attributes?